standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnQueries.java

```diff
@@ -18,36 +18,34 @@
 package org.apache.hadoop.hive.metastore.txn;
 
 public class TxnQueries {
-  public static final String SHOW_COMPACTION_ORDERBY_CLAUSE = 
-    " ORDER BY CASE " +
-    "   WHEN \"CC_END\" > \"CC_START\" and \"CC_END\" > \"CC_COMMIT_TIME\" " +
-    "     THEN \"CC_END\" " +
-    "   WHEN \"CC_START\" > \"CC_COMMIT_TIME\" " +
-    "     THEN \"CC_START\" " +
-    "   ELSE \"CC_COMMIT_TIME\" " +
+  public static final String SHOW_COMPACTION_ORDERBY_CLAUSE = "  ORDER BY  CASE " +
+    " WHEN \"CC_END\" > \"CC_START\" and \"CC_END\" > \"CC_COMMIT_TIME\" " +
+    " THEN \"CC_END\" " +
+    " WHEN \"CC_START\" > \"CC_COMMIT_TIME\" " +
+    " THEN \"CC_START\" " +
+    " ELSE \"CC_COMMIT_TIME\" " +
     " END desc ," +
     " \"CC_ENQUEUE_TIME\" asc";
 
-  public static final String SHOW_COMPACTION_QUERY = 
-    "SELECT XX.* FROM ( SELECT " +
-    "  \"CQ_DATABASE\" AS CC_DATABASE, \"CQ_TABLE\" AS CC_TABLE, \"CQ_PARTITION\" AS CC_PARTITION, " +
-    "  \"CQ_STATE\" AS CC_STATE, \"CQ_TYPE\" AS CC_TYPE, \"CQ_WORKER_ID\" AS CC_WORKER_ID, " +
-    "  \"CQ_START\" AS CC_START, -1 \"CC_END\", \"CQ_RUN_AS\" AS CC_RUN_AS, " +
-    "  \"CQ_HADOOP_JOB_ID\" AS CC_HADOOP_JOB_ID, \"CQ_ID\" AS CC_ID, \"CQ_ERROR_MESSAGE\" AS CC_ERROR_MESSAGE, " +
-    "  \"CQ_ENQUEUE_TIME\" AS CC_ENQUEUE_TIME, \"CQ_WORKER_VERSION\" AS CC_WORKER_VERSION, " +
-    "  \"CQ_INITIATOR_ID\" AS CC_INITIATOR_ID, \"CQ_INITIATOR_VERSION\" AS CC_INITIATOR_VERSION, " +
-    "  \"CQ_CLEANER_START\" AS CC_CLEANER_START, \"CQ_POOL_NAME\" AS CC_POOL_NAME, \"CQ_TXN_ID\" AS CC_TXN_ID, " +
-    "  \"CQ_NEXT_TXN_ID\" AS CC_NEXT_TXN_ID, \"CQ_COMMIT_TIME\" AS CC_COMMIT_TIME, " +
-    "  \"CQ_HIGHEST_WRITE_ID\" AS CC_HIGHEST_WRITE_ID " +
+  public static final String SHOW_COMPACTION_QUERY = "SELECT XX.* FROM ( SELECT " +
+    " \"CQ_DATABASE\" AS \"CC_DATABASE\", \"CQ_TABLE\" AS \"CC_TABLE\", \"CQ_PARTITION\" AS \"CC_PARTITION\", " +
+    " \"CQ_STATE\" AS \"CC_STATE\", \"CQ_TYPE\" AS \"CC_TYPE\", \"CQ_WORKER_ID\" AS \"CC_WORKER_ID\", " +
+    " \"CQ_START\" AS \"CC_START\", -1 AS \"CC_END\", \"CQ_RUN_AS\" AS \"CC_RUN_AS\", " +
+    " \"CQ_HADOOP_JOB_ID\" AS \"CC_HADOOP_JOB_ID\", \"CQ_ID\" AS \"CC_ID\", \"CQ_ERROR_MESSAGE\" AS \"CC_ERROR_MESSAGE\", " +
+    " \"CQ_ENQUEUE_TIME\" AS \"CC_ENQUEUE_TIME\", \"CQ_WORKER_VERSION\" AS \"CC_WORKER_VERSION\", " +
+    " \"CQ_INITIATOR_ID\" AS \"CC_INITIATOR_ID\", \"CQ_INITIATOR_VERSION\" AS \"CC_INITIATOR_VERSION\", " +
+    " \"CQ_CLEANER_START\" AS \"CC_CLEANER_START\", \"CQ_POOL_NAME\" AS \"CC_POOL_NAME\", \"CQ_TXN_ID\" AS \"CC_TXN_ID\", " +
+    " \"CQ_NEXT_TXN_ID\" AS \"CC_NEXT_TXN_ID\", \"CQ_COMMIT_TIME\" AS \"CC_COMMIT_TIME\", " +
+    " \"CQ_HIGHEST_WRITE_ID\" AS \"CC_HIGHEST_WRITE_ID\" " +
```

Comment:
I agree with @deniskuzZ . Even with ignoring whitespace changes, it is impossible to tell what is the difference.
<img width="1611" alt="image" src="https://user-images.githubusercontent.com/1486749/202659443-f38ec76d-ab8c-474a-83ad-83bae8fff5e1.png">

Reply:
Resolved